Changes for 1.5.x productized build of Operator bundle

almope opened the pull request:
Signed-off-by: Alberto Morales Perez <almorale@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

vaibhavjainwiz reviewed (commented):
You also need to make changes in at below as well:
https://github.com/kiegroup/rhpam-kogito-operator/blob/f875aa429933f8423493b786741ed91cb4274d9f/config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml#L8

almope commented:
> You also need to make changes in at below as well:
> https://github.com/kiegroup/rhpam-kogito-operator/blob/f875aa429933f8423493b786741ed91cb4274d9f/config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml#L8

Ok, changed that and fixed the Operator image version to `7.11.0`

vaibhavjainwiz commented:
Raised new PR to make it configurable 
https://github.com/kiegroup/rhpam-kogito-operator/pull/40
